scenarios/openshift/customapp_pod.yaml

```diff
@@ -1,32 +1,10 @@
-config:
-  runStrategy:
-    runs: 1
-    maxSecondsBetweenRuns: 30
-    minSecondsBetweenRuns: 1
-scenarios:
-  - name: "delete acme-air pods"
-    steps:
-    - podAction:
-        matches:
-          - labels:
-              namespace: "acme-air"
-              selector: ""
-        filters:
-          - randomSample:
-              size: 1
-        actions:
-          - kill:
-              probability: 1
-              force: true
-    - podAction:
-        matches:
-          - labels:
-              namespace: "acme-air"
-              selector: ""
-        retries:
-          retriesTimeout:
-            timeout: 180
-
-        actions:
-          - checkPodCount:
-              count: 8
+# yaml-language-server: $schema=../plugin.schema.json
+- id: kill-pod
+  config:
+    namespace_pattern: ^acme-air$
+    kill: 1
+- id: wait-for-pods
```

Comment:
Can the wait action be part of the kill-pod config to have everything in one place to avoid specifying the namespace pattern again? This will also help if we want to run the scenario for n number of times. Thoughts?